lib/symboldatabase.cpp

```diff
@@ -1439,6 +1439,10 @@ void SymbolDatabase::createSymbolDatabaseExprIds()
             } else if (Token::Match(tok, "(|.|[|%cop%")) {
                 exprs[tok->str()].push_back(tok);
                 tok->exprId(id++);
+
+                if (id == std::numeric_limits<nonneg MathLib::bigint>::max()) {
```

Comment:
this condition can't be true right? an int compared to the max value of long long?